Berksfile.lock

```diff
@@ -123,7 +142,7 @@ GRAPH
   python (1.4.6)
     build-essential (>= 0.0.0)
     yum-epel (>= 0.0.0)
-  rabbitmq (3.3.0)
+  rabbitmq (3.9.0)
```

Comment:
This version of the rabbitmq did not work on the version of chef opsworks runs when I tried it.